components/brave_new_tab_ui/components/default/binance/style.ts

```diff
@@ -0,0 +1,185 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+interface StyleProps {
+  hide?: boolean
+  itemsShowing?: boolean
+  isSmall?: boolean
+  isInTab?: boolean
+  isLast?: boolean
+  disabled?: boolean
+  isFiat?: boolean
+}
+
+export const WidgetWrapper = styled<{}, 'div'>('div')`
+  color: white;
+  padding: 10px 15px;
+  border-radius: 6px;
+  position: relative;
+  font-family: Muli, sans-serif;
+  overflow-x: hidden;
+  min-width: 284px;
+  background-image: linear-gradient(140deg, #1F2327 0%, #000000 100%);
+`
+
+export const Header = styled<StyleProps, 'div'>('div')`
+  margin-top: ${p => p.isInTab ? 0 : 10}px;
+  text-align: left;
+`
+
+export const Content = styled<{}, 'div'>('div')`
+  margin: 10px 0;
+  min-width: 255px;
+`
+
+export const Title = styled<{}, 'span'>('span')`
+  display: block;
+  font-size: 13px;
+  font-weight: bold;
+`
+
+export const Copy = styled<{}, 'p'>('p')`
+  font-size: 15px;
+  max-width: 240px;
+  margin-top: 20px;
+  margin-bottom: 11px;
+`
+
+export const BuyPromptWrapper = styled<{}, 'div'>('div')`
+  margin-bottom: 15px;
+`
+
+export const FiatInputWrapper = styled<{}, 'div'>('div')`
+  height: 30px;
+  border: 1px solid rgb(70, 70, 70);
+  margin-bottom: 10px;
+`
+
+export const FiatDropdown = styled<StyleProps, 'div'>('div')`
+  float: right;
+  width: 25%;
+  padding: 7px 3px 0px 7px;
+  cursor: ${p => p.disabled ? 'auto' : 'pointer'};
+`
+
+export const FiatDropdownLabel = styled<{}, 'span'>('span')`
+`
+
+export const CaratDropdown = styled<StyleProps, 'div'>('div')`
+  width: 14px;
+  height: 14px;
+  float: right;
+  color: #fff;
+  visibility: ${p => p.hide ? 'hidden' : 'visible'};
+`
+
+export const InputField = styled<{}, 'input'>('input')`
+  display: inline-block;
+  min-width: 215px;
+  height: 30px;
+  vertical-align: top;
+  border: none;
+  color: rgb(70, 70, 70);
+  background: #000;
+  border: 1px solid rgb(70, 70, 70);
+  border-left: none;
+  padding-left: 5px;
+  &:focus {
```

Comment:
nit: Please add a line break before this pseudo-class

Reply:
done